src/Sparrow/Json/BlittableJsonTextWriter.cs

```diff
@@ -94,52 +94,62 @@ public abstract unsafe class AbstractBlittableJsonTextWriter : IDisposable
         private const byte Comma = (byte)',';
         private const byte Quote = (byte)'"';
         private const byte Colon = (byte)':';
-        public static readonly byte[] NaNBuffer = { (byte)'"', (byte)'N', (byte)'a', (byte)'N', (byte)'"' };
-        public static readonly byte[] PositiveInfinityBuffer =
+        public static ReadOnlySpan<byte> NaNBuffer => new byte[] { (byte)'"', (byte)'N', (byte)'a', (byte)'N', (byte)'"' };
+        public static ReadOnlySpan<byte> PositiveInfinityBuffer => new byte[]
```

Comment:
This create _properties_,  which allocates a new value each time.
Pretty sure that isn't what you intended to do.

Reply:
Nope, those are subjected to a special optimization at the JIT level. What happens there is that those values are baked in at the dll .text section. It is a neat trick :D
https://vcsjones.dev/csharp-readonly-span-bytes-static/

Reply:
I remember reading that, but I though it was for span fields, not for properties